mod-orders-storage/examples/routing_list_collection.sample

```diff
@@ -1,7 +1,6 @@
 {
     "routingLists": [
       {
-        "id": "c0d13648-347b-4ac9-8c2f-5bc47248b87e",
         "_version": 1,
```

Comment:
in only post sample id field can be deleted

Reply:
Good catch, you can rereview now